linker/spm.jp0.lst

```diff
@@ -3,7 +3,126 @@
 800030c8:firstRel
 800030cc:lastRel
 
-// Automatically generated from spm.eu0.lst 
+80C58500:rpg_handle_menu
```

Comment:
The non-eu0 lsts should add all new symbols in batches to the bottom with a comment on where they're being added from